script.js

```diff
@@ -0,0 +1,107 @@
+var people={
```

Comment:
I might make people an array rather than an object.  That way you can loop through it more easily (since order doesn't matter anyway)